src/core/lib/iomgr/parse_address.cc

```diff
@@ -75,6 +75,7 @@ bool grpc_parse_unix_abstract(const grpc_uri* uri,
   // Abstract sockets paths begin with a null charcter.
   un->sun_path[0] = '\0';
   // The address length of an abstract unix socket does not include a terminating null.
+  // TODO(#24638): Support null characters in abstract socket names.
```

Comment:
Please include the full URI of the issue.

Reply:
Done